specs/vep-1493-vault-integration/diagrams/high_level_design.puml

```diff
@@ -0,0 +1,31 @@
+@startuml
+!include <awslib/AWSSimplified>
+!include <awslib/General/User>
+!includeurl https://raw.githubusercontent.com/plantuml-stdlib/gilbarbara-plantuml-sprites/v1.0/sprites/vault.puml
+
+User(admin, "Administrator", " ")
+User(engineer, "User", " ")
+
+rectangle "K8s Cluster" {
+   component "Control Service" as cs #teal {
+    rectangle "Properties " as properties_service #teal
+   }
+}
+
+rectangle "  <&cog*4>\nVDK SDK" as sdk #teal
+database "\nCockroachDB\n" as DB
+rectangle "<$vault>\n  Vault " as vault #yellow
+
+engineer -- sdk : get/set property\nfor a data job
+sdk -- properties_service : API call
```

Comment:
You may indicate auth. in some ways. Maybe `API Call (User Oauth2 token)`

Reply:
done.